src/devices/SensorHub/SensorHub.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.SensorHub
+{
+    /// <summary>
+    /// SensorHub
+    /// </summary>
+    public class SensorHub
+    {
+        /// <summary>
+        /// Default I2C bus id
+        /// </summary>
+        public const int DefaultI2cBusId = 1;
+
+        /// <summary>
+        /// Default I2C address
+        /// </summary>
+        public const int DefaultI2cAddress = 0x17;
+
+        private readonly I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Creates a SensorHub with the default bus and connection values
+        /// </summary>
+        /// <returns>SensorHub</returns>
+        public static SensorHub Create()
+        {
+            return new(I2cDevice.Create(new I2cConnectionSettings(DefaultI2cBusId, DefaultI2cAddress)));
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SensorHub"/> class.
+        /// </summary>
+        public SensorHub(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+        }
+
+        /// <summary>
+        /// Try to read the temperature from the off board thermometer in the range -30 to 127 Celsius
+        /// </summary>
+        /// <param name="temperature">The temperature in Celsius if successful</param>
```

Comment:
The comment should not say "in Celsius" if the method is returning an instance of `Temperature` since that caries the unit. (similar in all of these methods).

Reply:
In other words, it's ok to give the typical range, just don't write it will return Celsius. Example:
```
<summary>Try to read the temperature from the off board thermometer. </summary>
<remarks>Range is -30 to 127 Celsius.</remarks>
<param name="temperature">The temperature if successful</param>
```

Reply:
done